src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.styles.ts

```diff
@@ -0,0 +1,71 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+import { ellipsisTextStyle } from '~/utils/helper-functions'
+
+export const style = {
+  header: {
+    typography: TypographyVariantEnum.H6,
+    color: palette.basic.blueGray,
+    ml: '20px',
+    mb: '30px'
```

Comment:
can you use a single m property or do you need to rewrite these margins?

Reply:
yes, done